From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Thu, 15 Dec 2022 12:19:04 +0100
Subject: [PATCH] don't try to generate manpages

The script tries to execute mksquashfs which does not work for
cross-compiling.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 squashfs-tools/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/squashfs-tools/Makefile b/squashfs-tools/Makefile
index 8b80b8a68c1b..3d147735f506 100755
--- a/squashfs-tools/Makefile
+++ b/squashfs-tools/Makefile
@@ -422,4 +422,3 @@ install: mksquashfs unsquashfs
 	cp unsquashfs $(INSTALL_DIR)
 	ln -fs unsquashfs $(INSTALL_DIR)/sqfscat
 	ln -fs mksquashfs $(INSTALL_DIR)/sqfstar
-	../generate-manpages/install-manpages.sh $(shell pwd)/.. "$(INSTALL_MANPAGES_DIR)"
